userspace/libsinsp/chisel_api.cpp

```diff
@@ -111,6 +111,7 @@ uint32_t lua_cbacks::rawval_to_lua_stack(lua_State *ls, uint8_t* rawval, ppm_par
 			return 1;
 		case PT_CHARBUF:
 		case PT_FSPATH:
+	    case PT_FSRELPATH:
```

Comment:
Indentation (in several places across this PR) :)

Reply:
Thanks for pointing it out. My editor settings are very opinionated as to indentation and don't deal well with files that have opposite opinions. I'll fix it.